src/Auth0.OidcClient.Core/Auth0ClientBase.cs

```diff
@@ -19,37 +19,30 @@ public abstract class Auth0ClientBase : IAuth0Client
     {
         private readonly Auth0ClientOptions _options;
         private readonly string _userAgent;
+        private readonly bool _formPost;
         private IdentityModel.OidcClient.OidcClient _oidcClient;
 
         /// <summary>
         /// Creates a new instance of the Auth0 OIDC Client.
         /// </summary>
         /// <param name="options">The <see cref="Auth0ClientOptions"/> specifying the configuration for the Auth0 OIDC Client.</param>
         /// <param name="platformName">The platform name that forms part of the user-agent when communicating with Auth0 servers.</param>
-        /// <param name="responseMode">Optional <see cref="AuthorizeResponseMode"/> authorization should operate in, defaults to <see cref="AuthorizeResponseMode.Redirect"/></param>
-        public Auth0ClientBase(Auth0ClientOptions options, string platformName, AuthorizeResponseMode responseMode = AuthorizeResponseMode.Redirect)
+        /// <param name="formPost">Optionally whether to use a form post instead of a redirect using authentication.</param>
```

Comment:
Just a note ... might want to look into whether this should be deprecated or not. If all the devices that use this library can use a browser, then we don't want to offer RO here.